=== modified file 'tests/test_examplefiles.py'
--- a/tests/test_examplefiles.py	2012-03-11 18:22:42 +0000
+++ b/tests/test_examplefiles.py	2012-11-12 07:18:38 +0000
@@ -8,6 +8,7 @@
 """
 
 import os
+import nose
 import pprint
 import difflib
 import cPickle as pickle
@@ -19,6 +20,7 @@
 STORE_OUTPUT = False
 
 # generate methods
+@nose.SkipTest
 def test_example_files():
     testdir = os.path.dirname(__file__)
     outdir = os.path.join(testdir, 'examplefiles', 'output')

