Index: pygments-0.9/setup.py
===================================================================
--- pygments-0.9.orig/setup.py	2007-12-10 19:54:59.000000000 +0100
+++ pygments-0.9/setup.py	2007-12-10 19:55:06.000000000 +0100
@@ -20,11 +20,11 @@
     keywords = 'syntax highlighting',
     packages = find_packages(),
     scripts = ['pygmentize'],
-    entry_points = {
-        'console_scripts': [
-            'pygmentize = pygments.cmdline:main',
-        ],
-    },
+#    entry_points = {
+#        'console_scripts': [
+#            'pygmentize = pygments.cmdline:main',
+#        ],
+#    },
     platforms = 'any',
     zip_safe = False,
     include_package_data = True,
